constraint-evaluation-generator/src/main.rs

```diff
@@ -0,0 +1,325 @@
+use std::collections::HashSet;
+
+use itertools::Itertools;
+use triton_vm::table::base_table::InheritsFromTable;
+use triton_vm::table::challenges::TableChallenges;
+use triton_vm::table::constraint_circuit::{
+    CircuitExpression, CircuitId, ConstraintCircuit, InputIndicator,
+};
+use triton_vm::table::instruction_table::ExtInstructionTable;
+use triton_vm::table::jump_stack_table::ExtJumpStackTable;
+use triton_vm::table::op_stack_table::ExtOpStackTable;
+use triton_vm::table::processor_table::ExtProcessorTable;
+use triton_vm::table::program_table::ExtProgramTable;
+use triton_vm::table::ram_table::ExtRamTable;
+use twenty_first::shared_math::b_field_element::BFieldElement;
+use twenty_first::shared_math::x_field_element::XFieldElement;
+
+fn main() {
+    println!("Generate those constraint evaluators!");
+
+    // Program table
+    let table = ExtProgramTable::default();
+    let mut constraint_circuits = ExtProgramTable::ext_transition_constraints_as_circuits();
+    gen(
+        &table,
+        "program_table",
+        "ProgramTable",
```

Comment:
I feel like it's cleaner to generate the one from the other, or both from an object like `vec!["program"]` or `vec!["jump", "stack"]`.